node/network/availability-distribution/src/lib.rs

```diff
@@ -836,7 +817,7 @@ async fn query_live_candidates<Context>(
 	ctx: &mut Context,
 	state: &mut ProtocolState,
 	relay_parents: impl IntoIterator<Item = Hash>,
-) -> Result<HashMap<Hash, (CandidateHash, CommittedCandidateReceipt)>>
+) -> Result<HashMap<(Hash, CandidateHash), CommittedCandidateReceipt>>
```

Comment:
I would expect this to be either `HashMap<Hash, Vec<(CandidateHash, CommittedCandidateReceipt)>>` or `HashMap<CandidateHash, CommittedCandidateRecipt>`. Note that with the first variant, as well as what is currently committed in this PR, the same candidate is likely to appear more than once, if it's been pending availability for more than one block.

I'm not sure where the relay-parent is supposed to enter into this.

To be honest, I don't really understand where the relay-parent needs to fit in here.